frontend/src/components/ActionButton.tsx

```diff
@@ -1,12 +1,15 @@
 import React, { ReactNode } from 'react';
+import { Tooltip } from 'react-tooltip';
+import { tooltipStyle } from '../lib/constants';
 
 interface ActionButtonProps {
   link?: string;
   onClick?: () => void;
+  tooltipLabel?: string;
   children: ReactNode;
 }
 
-const ActionButton: React.FC<ActionButtonProps> = ({ link, onClick, children }) => {
+const ActionButton: React.FC<ActionButtonProps> = ({ link, onClick, tooltipLabel, children }) => {
```

Comment:
Here and in other places:
```suggestion
const ActionButton: React.FC<ActionButtonProps> = ({ url, onClick, tooltipLabel, children }) => {
```